fetch/src/3_visualize.r

```diff
@@ -0,0 +1,45 @@
+plot_data <- function() {
```

Comment:
Same goes for this script - according to our conventions this would be saved in a subdirectory of the 3_visualize step: `3_vizualize/src`